hax/hax/util.py

```diff
@@ -589,6 +589,62 @@ def get_node_health(self, node: str) -> str:
             raise HAConsistencyException(
                 f'Failed to get {node} node health') from e
 
+    def get_node_fid(self, node: str) -> Optional[str]:
```

Comment:
In our code we already have an object representation for fids - `Fid` class. Please use it instead of raw strings.
Reason: types can narrow down the ways how the code can be broken. Typed code is more explicit and more clear for static analyzers.

And please also note that the function can raise `HAConsistencyException`. Don't you need `@repeat_if_fails` decorator?

Reply:
here, the function param `node` is actually the hostname of node (like eg. `"ssc-vm-c-1899.colo.seagate.com"`). So I have to use it as a string. 
however, the function retrieves the fid of the node. **I will change it to return `Fid`.** 

> And please also note that the function can raise HAConsistencyException. Don't you need @repeat_if_fails decorator?

Yes, will add it. Thanks.

Reply:
Done